panther_driver/src/driver_node.py

```diff
@@ -384,6 +248,19 @@ def driverNode():
 
             joint_state_publisher.publish(joint_state_msg)
 
+            # read drivers battery data
+            try:
+                battery_driver_msg.V_front = float(front_controller.sdo[0x210D][2].raw)/10
+                battery_driver_msg.V_rear = float(rear_controller.sdo[0x210D][2].raw)/10
```

Comment:
In my opinion, it is better to use the same way of addressing in each command. ParameterName (Qry_VOLTS) seems more readable than CanOpen ID (0x210D).

Reply:
Good point. I've changed that